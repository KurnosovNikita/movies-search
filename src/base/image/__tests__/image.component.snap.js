import React from 'react';
import {shallow} from 'enzyme';
import Image from '../image.component';

describe('Image component', () => {
    let props;
    beforeEach(() => {
        props = {
            onClick: jest.fn(),
            className: 'className',
            src: 'src',
            imageAlt: 'imageAlt',
        };
    });

    it('should render <Image> correctly', () => {
        const component = shallow(<Image {...props}/>);

        expect(component).toMatchSnapshot();
    });

    describe('render props correctly', () => {

        it('render with className', () => {
            const component = shallow(<Image {...props}/>);

            expect(component.props().className).toEqual('className');
        });

        it('render with src', () => {
            const component = shallow(<Image {...props}/>);

            expect(component.props().src).toEqual('src');
        });
    });
});
