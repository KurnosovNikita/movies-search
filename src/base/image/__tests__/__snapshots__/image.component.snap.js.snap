// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component should render <Image> correctly 1`] = `
<img
  alt="imageAlt"
  className="className"
  onClick={[MockFunction]}
  src="src"
/>
`;
